docs/test-v1/smoke-test-3.md

```
@@ -0,0 +1,60 @@
+---
+
+## 🔴 Still Broken (0)
+
+All issues from Round 2 have been resolved.
+
+---
+
+## 🆕 New Issues Found (5)
+
+### ✅ Tasks / 👥 Members
+| # | Issue | Type | Device | GitHub Issue |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)